apps/server/src/utils/parser.ts

```diff
@@ -149,6 +151,8 @@ export const parseExcel = (excelData: unknown[][], options?: Partial<ExcelImport
         [importMap.user7]: (index: number) => (user7Index = index),
         [importMap.user8]: (index: number) => (user8Index = index),
         [importMap.user9]: (index: number) => (user9Index = index),
+        [importMap.timeWarning]: (index: number) => (timeWarningIndex = index),
+        [importMap.timeDanger]: (index: number) => (timeDangerIndex = index),
```

Comment:
nice! does this means we also need to update the components related to the sheet import?

Reply:
If i'm not mistaken no ... I have tested importing and exporting rundowns and everything seems to be working fine.

Reply:
what I am calling the sheet import is the import of excel xlsx, you need to give the columns to import.
I havent seen this defined anywhere or added to the preview items

Reply:
will be solved in a separate pr